numpy/lib/utils.py

```diff
@@ -193,7 +179,40 @@ def deprecate(*args, **kwargs):
     else:
         return _Deprecate(*args, **kwargs)
 
-deprecate_with_doc = lambda msg: _Deprecate(message=msg)
+def deprecate_with_doc(msg):
+    """
+    Returns new object. 
```

Comment:
Need a blank line here.

Reply:
Also, please make this a bit more descriptive. Something like:
```
Decorator to deprecate a function and change its docstring to mention the deprecation
```

Reply:
Thanks @rgommers . I have made the required change.